test/e2e/framework/lifecycle/upgrade.go

```diff
@@ -26,39 +26,40 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apimachinery/pkg/version"
 	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
```

Comment:
Can you use `Logf` from `test/e2e/framework/log`?

Reply:
Yup just see your issue. I'll fix that. Thanks

Reply:
Fixed